From: Andrew Morton <akpm@linux-foundation.org>
Subject: lib-stackdepot-mm-rename-stack_depot_want_early_init-fix
Date: Mon Jan 30 05:43:08 PM PST 2023

fix it for "mm: use stack_depot_early_init for kmemleak"

Cc: Alexander Potapenko <glider@google.com>
Cc: Andrey Konovalov <andreyknvl@google.com>
Cc: Evgenii Stepanov <eugenis@google.com>
Cc: Marco Elver <elver@google.com>
Cc: Vlastimil Babka <vbabka@suse.cz>
Cc: Zhaoyang Huang <zhaoyang.huang@unisoc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/kmemleak.c~lib-stackdepot-mm-rename-stack_depot_want_early_init-fix
+++ a/mm/kmemleak.c
@@ -105,6 +105,9 @@
 #include <linux/kmemleak.h>
 #include <linux/memory_hotplug.h>
 
+/* akpm: stupid temporary hack */
+#define stack_depot_want_early_init stack_depot_request_early_init
+
 /*
  * Kmemleak configuration and common defines.
  */
_
